portal/views/assessment_engine.py

```diff
@@ -1788,12 +1794,14 @@ def patient_assessment_status(patient_id):
     patient = get_user_or_abort(patient_id)
     current_user().check_role(permission='view', other_id=patient_id)
 
-    now = datetime.utcnow()
+    date = request.args.get('as_of_date')
+    # allow date and time info to be available
+    date = FHIR_datetime.parse(date) if date else datetime.utcnow()
```

Comment:
Why not set (and pass) to `None`, so `QB_Status()` always sets the default?

Reply:
oh, clarity.  no reason.

I'll change, and recreate the PR to hit the RC.